views/guide.lavender

```diff
@@ -89,8 +89,9 @@ block content
           h2#github via Github      
           p
             | Add the lavender submodule with 
-            code git submodule add git@github.com:golavender/lavender.git (folder to 
-            | clone to) and include it with 
+            code git submodule add git@github.com:golavender/lavender.git (folder to
+            code clone to)
```

Comment:
You shouldn't use `code` on each line. Instead you can keep it basically how it was before. Just be sure to indent the second line.

```
code git submodule add git@github.com:golavender/lavender.git (folder to
  | clone to)[
| and include it with 
```